src/protocol/payload/block.rs

```diff
@@ -591,6 +591,54 @@ mod tests {
         assert_eq!(block_bytes, buffer);
     }
 
+    #[test]
+    #[ignore]
```

Comment:
why are these ignored? doesn't seem like something slow

Reply:
All unit tests are ignored currently, so we can distinguish them from the Ziggurat suite and run the latter with `cargo test`.